tests/unit_tests/test_platzky.py

```
@@ -0,0 +1,91 @@
+    def mock_db(self) -> MagicMock:
+        return MagicMock()
+
+    def test_change_language_with_domain(self, mock_db):
+        """Test the change_language function when a domain is specified."""
+        # Mock the languages
+        mock_config = MagicMock()
+        mock_config.languages = {
+            "en": LanguageConfig(name="English", flag="gb", country="GB", domain="example.com"),
+            "de": LanguageConfig(name="German", flag="de", country="DE", domain="example.de")
+        }
+        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)